server/sonar-server/src/main/java/org/sonar/server/component/ws/FilterParser.java

```diff
@@ -34,10 +34,12 @@
 
 public class FilterParser {
 
+  private static final String QUOTE = "'";
```

Comment:
I prefer to use only one character, it allows to use double quote in the search (note that we could do the reverse : use double only quotes)